code/game/world.dm

```diff
@@ -335,28 +332,21 @@ GLOBAL_VAR(restart_counter)
 
 	s += "<b>[station_name()]</b>";
 	var/discordurl = CONFIG_GET(string/discordurl)
-	s += "(<a href='[discordurl]'>Discord</a>|<a href='http://beestation13.com'>Website</a>))"
+	s += " (<a href='[discordurl]'>Discord</a>|<a href='http://beestation13.com'>Website</a>))"
 
 	var/players = GLOB.clients.len
 
-	var/popcaptext = ""
-	var/popcap = max(CONFIG_GET(number/extreme_popcap), CONFIG_GET(number/hard_popcap), CONFIG_GET(number/soft_popcap))
-	if (popcap)
-		popcaptext = "/[popcap]"
-
-	if (players > 1)
-		features += "[players][popcaptext] players"
-	else if (players > 0)
-		features += "[players][popcaptext] player"
-
 	game_state = (CONFIG_GET(number/extreme_popcap) && players >= CONFIG_GET(number/extreme_popcap)) //tells the hub if we are full
 
 	if (!host && hostedby)
 		features += "hosted by <b>[hostedby]</b>"
 
-	if (features)
+	if(length(features))
 		s += ": [jointext(features, ", ")]"
 
+	s += "<br>Time: <b>[gameTimestamp("hh:mm")]</b>"
+	s += "<br>Alert: <b>[capitalize(get_security_level())]</b>"
```

Comment:
I think players & popcap should be included in here (Useful to know if the server is full and how much space is left)

Reply:
@PowerfulBacon done

Reply:
Popcap is already included. It's a basic hub feature. 
![image](https://user-images.githubusercontent.com/26130695/181068950-3a0962d2-c0c7-4793-a019-0ff9f54940e4.png)

Reply:
Disregard, misunderstood the request